packages/extension-table/src/utilities/colStyle.ts

```diff
@@ -0,0 +1,14 @@
+export function getColStyleDeclaration(minWidth: number, width: number | undefined): [string, string] {
+  if (width) {
+    // apply the stored width unless it is below the configured minimum cell width
+    return ['width', `${Math.max(width, minWidth)}px`]
+  }
+
+  // set the minimum with on the column if it has no stored width
+  return ['min-width', `${minWidth}px`]
+
+}
+
+export function getColStyle(minWidth: number, width: number): string {
+  return getColStyleDeclaration(minWidth, width).join(': ')
+}
```

Comment:
This is abstracting over a couple of if statements, I'm not sure it warrants the complexity it adds.

I would honestly prefer for their to be a single function here, either stick to the string version or the array version, both just adds complexity and indirection.

Reply:
I extracted them because the logic was originally copy-pasted in both the create and the update code, I can remove the `getColStyle` function and do the concatenation in the table creation function directly instead of having a function in this file